pkgs/development/libraries/allegro/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, texinfo6_5, libXext, xorgproto, libX11
+{ lib, stdenv, fetchurl, texinfo_6_5, libXext, xorgproto, libX11
```

Comment:
Recently I was discussing about a better overriding interface. Instead of using explicit `libXYZ_versionString`, use `libXYZ` and set its value at top-level (usually all-packages.nix).

Reply:
IMHO, when no default is preferred, a better pattern is `{..., texinfo_6_5, texinfo ? texinfo_6_5, ...}`. If some version is a prefered default, then the way it is done here actually seems the best to me, since those overrides in `all-packages.nix` are annoying and need to be kept in sync with the expression.

Reply:
I thought about the approach you propose too, @AndersonTorres, yet came to the conclusion that package preferring a specific version do so in their package definition rather than in `all-packages.nix` to avoid the noise in that file and avoid the need to keep it in sync as @oxij put it

Reply:
Is `texinfo ? texinfo_6_5` compatible with having a default top-level texinfo version though?

Reply:
No.